src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -187,6 +188,13 @@ def main(
         num_jobs=len(job_parameters),
         redis_logging_root=output_paths.logging_root,
     )
+    # Spin up a unified interface to all the redis databases
+    registry_manager = redis_dbs.RegistryManager(redis_ports, num_jobs_completed)
+    # Distribute all the remaining jobs across the job queues
+    # in the redis databases.
+    registry_manager.enqueue(
+        jobs=job_parameters, workhorse_import_path=worker.WORK_HORSE_PATHS["vivarium"]
```

Comment:
Nit: can "vivarium" ultimately become a constant?

Reply:
For sure.  It's in a lot of places and I don't remember the context for all of them.  I plan to address in the load_test jobs bucket of work.